app/db/models/index.js

```diff
@@ -506,24 +497,20 @@ module.exports = function (sequelize, env) {
     },
 
     compare: function (pass, user, callback) {
-      bcrypt.compare(pass, user.password.saltedHash, function (err, res) {
-        callback(err, res);
-      });
+      bcrypt.compare(pass, user.Password.saltedHash, callback);
     },
 
     createOauthLogin: function (userId, clientId, callback) {
       oauthClient.findOrCreate({
-        client: clientId
-      }, {
-        client: clientId
-      }).then(function (client) {
-        return oauthLogin.create({
+          client: clientId
```

Comment:
according to http://docs.sequelizejs.com/manual/tutorial/models-usage.html#-findorcreate-search-for-a-specific-element-or-create-it-if-not-available, shouldn't you only be able to pass a single object into `findOrCreate`?
```js
{
  where: {
    client: clientId
  },
  defaults: {
    client: clientId
  }
}
```